TwitchLib.Client.Models/Extractors/EmoteExtractor.cs

```diff
@@ -51,22 +51,15 @@ public IEnumerable<Emote> Extract(string rawEmoteSetString, string message)
             }
         }
 
-        private Emote GetEmote(string emoteData, string emoteId, string message, bool single = false)
+        private static Emote GetEmote(string emoteData, string emoteId, string message, bool single = false)
         {
-            int startIndex = -1;
-            int endIndex = -1;
-
             if (single)
             {
-                startIndex = int.Parse(emoteData.Split(':')[1].Split('-')[0]);
-                endIndex = int.Parse(emoteData.Split(':')[1].Split('-')[1]);
+                emoteData = emoteData.Split(':')[1];
             }
-            else
-            {
-                startIndex = int.Parse(emoteData.Split('-')[0]);
-                endIndex = int.Parse(emoteData.Split('-')[1]);
-            }
-
+            var splitData = emoteData.Split('-');
+            var startIndex = int.Parse(splitData[0]);
```

Comment:
In case of splitData[0] being invalid the entire client crashes. TryParse is safer variant

Reply:
I don't think it's worth fixing (I also don't check if the array has enough elements  before I index it)